src/mmrm.cpp

```diff
@@ -33,11 +33,13 @@ Type objective_function<Type>::operator() ()
   DATA_VECTOR(y_vector);           // Response vector (length n).
   DATA_VECTOR(weights_vector);     // Weights vector (length n).
   DATA_IVECTOR(visits_zero_inds);  // Zero-based Visits vector (length n).
+  DATA_MATRIX(coordinates);        // Coordinates matrix.
   DATA_INTEGER(n_visits);          // Number of visits, which is the dimension of the covariance matrix.
   DATA_INTEGER(n_subjects);        // Number of subjects.
   DATA_IVECTOR(subject_zero_inds); // Starting indices for each subject (0-based) (length n_subjects).
   DATA_IVECTOR(subject_n_visits);  // Number of observed visits for each subject (length n_subjects).
   DATA_STRING(cov_type);           // Covariance type name.
+  DATA_INTEGER(spatial);           // Spatial covariance (1)? Otherwise non-spatial covariance.
```

Comment:
```suggestion
  DATA_INTEGER(is_spatial_int);           // Spatial covariance (1)? Otherwise non-spatial covariance.
```
and then make it a bool below called `is_spatial`